internal/pkg/bulk/opApiKey.go

```diff
@@ -5,15 +5,37 @@
 package bulk
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"math"
 
 	"github.com/elastic/fleet-server/v7/internal/pkg/apikey"
+	"github.com/elastic/fleet-server/v7/internal/pkg/es"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	expectedAPIKeySize = 64 // 64B
```

Comment:
Where do these numbers come from? How would we know if they need to change?

Reply:
i took api key and added some buffer. 
but knowing api key size i can do this dynamically, good point